Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -756,8 +788,25 @@ public Inventory[] AcceptsForStorage()
     /// <summary>
     /// Deconstructs the furniture.
     /// </summary>
-    public void Deconstruct()
+    public void SetDeconstructJob()
     {
+        Jobs.CancelAll();
```

Comment:
This will cancel current job but not prevent new job to be created from a lua call for instance.

Could we set a flag, like "public bool RequestedDeconstruction {get;set;}" to true and maybe disable all lua call (I know this will disable animation but...)

Any ideas?